Vizzuality/trase
# Feature/finish footer

## Asana

https://app.asana.com/0/0/1199085547430325/f

## Description

- Do some style changes to better fit the designs
- Create the dropdown functionality on the form
- Fix the form warnings and make it work

## Testing instructions

Try both forms with ENABLE_FOOTER_REDESIGN=true and check the designs on the asana task
